jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    clearMocks: true,
+    moduleFileExtensions: ['js', 'ts'],
+    testEnvironment: 'node',
+    testMatch: ['**/*.test.ts'],
+    transform: {
+      '^.+\\.ts$': 'ts-jest'
```

Comment:
Should we use `tsc` here, as that is what we are going to use for compiling the actual files?

Reply:
Did some reading. `ts-jest` is **required** for supporting jest?

Reply:
its used for 'mocked'